lib/nexpose/evidence.rb

```diff
@@ -0,0 +1,25 @@
+module Nexpose
+  class Evidence
+    def initialize(xml_node)
+      @xml = xml_node
+    end
+
+    def supported_tags
+      [:details]
+    end
+
+    def respond_to?(method, include_private=false)
```

Comment:
We're not using it, but it's there because we're just overriding the `respond_to?` method.